releases/kiam.yaml

```diff
@@ -12,13 +12,33 @@ releases:
 ## kiam - AWS Assumed Roles for Pods #######################################
 ################################################################################
 #
+# This release REQUIRES that kiam support RollingUpdate wihtout a service outage, because we have enabled
+# automatic rotation of the TLS secrets, and without RollingUpdate support that rotation will cause a service outage.
+#
+# Because kiam does not currently support RollingUpdate while also managing the iptable rule it requires
+# (see kiam issue 202 https://github.com/uswitch/kiam/issues/202 ), this release REQUIRES that you set up
+# the necessary iptable rule on every node using some other mechanism.
+# Cloudposse currently installs this iptable via a kops cluster hook, but you are free to install it however you like.
+#
+# The required iptable rule is
+#    PREROUTING -d 169.254.169.254/32 -i $INTERFACE_NAME -p tcp -m tcp --dport 80 -j DNAT --to-destination $HOST_LOCAL_IP:8181
+# where $INTERFACE_NAME is the pod networking interface name and HOST_LOCAL_IP is the node's local IPv4 address.
+#
```

Comment:
How we will run ip tables commands in case of autoscaling?

Reply:
It is run as a Cluster hook when creating `node`s